rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+wrap_comments = true
+# comment_width = 100
+# max_width = 100
+# imports_granularity = "Item"
+# group_imports =  "One"
+# edition = "2021"
```

Comment:
❓Can we enable these?

Reply:
oops! I temporarily disabled them as I was debugging problems with my local `rustfmt` which was hanging. 
fixed :)